src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,48 +3,149 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+
+    private int size;
+    private Node<T> first;
+    private Node<T> last;
+
     @Override
     public boolean add(T value) {
-        return false;
+        Node<T> newNode;
+        if (size == 0) {
+            newNode = new Node<>(value, null, null);
+            first = newNode;
+        } else {
+            newNode = new Node<>(value, last, null);
+            last.next = newNode;
```

Comment:
```suggestion
        Node<T> newNode = new Node<>(value, last, null);
        if (size == 0) {
            first = newNode;
        } else {
            last.next = newNode;
```
you can simplify your code a bit

Reply:
corrected